cmd/integration_test/build/buildspecs/conformance-eks-a-cli.yml

```diff
@@ -5,6 +5,7 @@ env:
     INTEGRATION_TEST_MAX_EC2_COUNT: 25
     T_VSPHERE_CIDR: "198.18.0.0/16"
     T_VSPHERE_PRIVATE_NETWORK_CIDR: "10.1.0.0/24"
+    SKIPPED_TESTS: "TestTinkerbellKubernetes120SimpleFlow,TestTinkerbellKubernetes121SimpleFlow"
```

Comment:
Skipping Tinkerbell tests in CI until lab is ready. Until then tests are to be ran locally by developers against the port99 lab.

Reply:
Actually we don’t need to skip here. So the tests run here are compiled with different a build tag and hence only the tests added here are run.
https://github.com/aws/eks-anywhere/blob/main/test/e2e/conformance_test.go#L1